src/emailpro/emailpro-error-messages.popover.html

```diff
@@ -1,5 +1,5 @@
 <ul class="m-0">
     <li data-ng-repeat="message in messageDetails">
```

Comment:
I think you could add `track by message.id`.

Reply:
Good idea, except that `message.id` is sometimes missing, so I used `track by $index` instead